packages/eip1193-permission-middleware/src/wallet-getPermissions.test.ts

```diff
@@ -65,7 +65,7 @@ const createMockedHandler = () => {
     }),
   );
   const getAccounts = jest.fn().mockReturnValue([]);
-  const response: PendingJsonRpcResponse<Json> = {
+  const response: PendingJsonRpcResponse = {
```

Comment:
Most of those changes are required because of `@metamask/utils` dedupe. The default type parameter for those is `Json` already, resulting in a `eslint` warning if we specify it.